docs/integrate/rsyslog/tutorial.md

```diff
@@ -0,0 +1,140 @@
+(rsyslog-tutorial)=
+# Store server logs on CrateDB for fast search and aggregations
+
+## Introduction
+
+CrateDB stores server logs efficiently and makes them easy to query.
+
+Common pain points with traditional log stacks and SIEMs include:
+
+* timeouts when searching across long time ranges
+* proprietary, complex query syntaxes
+* awkward integrations with application monitoring dashboards
+
+CrateDB addresses these issues: query logs with standard SQL from any
+PostgreSQL‑compatible tool, and use full‑text search and aggregations
+backed by efficient indexes. The sections below walk through a minimal
+setup.
+
+## Setup
+
+### CrateDB
+
+First, start CrateDB. For production, use a dedicated cluster. For this demo, run a single‑node container:
+
+```bash
+sudo docker run -d --name cratedb \
+  -p 4200:4200 -p 5432:5432 \
+  -e CRATE_HEAP_SIZE=1g \
+  crate:5.6.0 -Cdiscovery.type=single-node
+```
+
+Next, create a table for logs. Open `http://localhost:4200/#!/console` and run:
+
+```sql
+CREATE TABLE doc.systemevents (
+  message TEXT,
+  INDEX message_ft USING FULLTEXT(message) WITH (analyzer = 'english'),
+  facility INTEGER,
+  fromhost TEXT,
+  priority INTEGER,
+  DeviceReportedTime TIMESTAMP,
+  ReceivedAt TIMESTAMP,
+  InfoUnitID INTEGER,
+  SysLogTag TEXT
+);
+```
+Tip: On headless systems, run queries with the {ref}`command-line tools <connect-cli>`.
+
+Then we need an account for the logging system:
+
+```sql
+-- Use a strong secret; e.g. from a secret manager or env var.
+CREATE USER rsyslog WITH (PASSWORD='pwd123');
+```
+
+and we need to grant permissions on the table above:
+
+```sql
+GRANT DML ON TABLE doc.systemevents TO rsyslog;
+```
+
+### rsyslog
+
+We will use [rsyslog](https://github.com/rsyslog/rsyslog) to send the logs to CrateDB, for this setup we need `rsyslog` v8.2202 or higher and the `ompgsql` module:
+
+```bash
+sudo add-apt-repository -y ppa:adiscon/v8-stable
```

Comment:
I think `add-apt-repository` has also been phased out on modern Debian/Ubuntu? Shall we exercise and verify this explicitly? Did you?

Reply:
Yeah, it has become too complicated 😢